### DISCUSSION TITLE
Unoptimized works for next build but complains with missing width for next dev

### DISCUSSION THREAD

**Discussion by gregorfelkar** [vercel/next.js#44380] [Help] [answered]
i have set image: unoptimized: true in next.config.js:

```
const nextConfig = {
  webpack: (config, { webpack }) => {
    config.plugins.push(
      new webpack.IgnorePlugin({
        resourceRegExp: /^electron$/,
      })
    );
    return config;
  },
  images: {
    unoptimized: true,
  },
  reactStrictMode: true,
  swcMinify: true
};
```

I also added unoptimized prop:
```
<Image
  src="/images/vector.svg"
  alt="text"
  unoptimized={true}
/>
```

And when i run `npm run build` which translates to `next build` it works ok. But if i try to run it in dev mode `npm run dev` (`next dev`) it complaints with `Error: Image with src "/images/vector.svg" is missing required "width" property.`

I am using Nextjs v13 and this issue is driving me crazy since if i want to deploy i cannot develop and vice versa.

So my question is how can i either:
- Disable use of Image component (Image optimization) alltogether (i use few svg images and i'm also ok with <img> tag) so it wont complain in dev or prod
- Make it work in dev mode with unoptimized set to true

Thanks in advance.

**Comment by dzienisz** [comment, marked as answer]
add props `fill={true}`

**Reply by ogheneovo12** [reply]
added this and still get same error
